Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebView.cs

```diff
@@ -0,0 +1,1036 @@
+            Verify.IsApartmentState(ApartmentState.STA);
+
+            // TODO: Assign Feature Control Keys
+
+            // We use this map to lookup which invalidator method to call when the parent's properties change
+            InvalidatorMap[VisibilityProperty] = new PropertyInvalidator(OnVisibilityInvalidated);
+            InvalidatorMap[IsEnabledProperty] = new PropertyInvalidator(OnIsEnabledInvalidated);
+#pragma warning restore 1065
+        }
+
+        private bool WebViewInitialized => _initializationState == InitializationState.IsInitialized;
+
+        private bool WebViewInitializing => _initializationState == InitializationState.IsInitializing;
+
+        private bool WebViewControlInitialized => _webViewControl != null;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebView"/> class.
+        /// </summary>
+        [SecurityCritical]
+        public WebView()
+        {
+            // TODO: Check whether browser is disabled
+            // TODO: Handle case (OnLoad) for handling POPUP windows
+            _initializationComplete = new ManualResetEvent(false);
+        }
+
+        internal WebView(WebViewControlHost webViewControl)
+            : this()
+        {
+            _webViewControl = webViewControl ?? throw new ArgumentNullException(nameof(webViewControl));
+            _process = webViewControl.Process;
+        }
+
+        /// <inheritdoc />
+        public override void BeginInit()
+        {
+            if (WebViewInitialized)
+            {
+                // Cannot initialize WebView since it is already completely initialized
+                throw new InvalidOperationException(DesignerUI.E_WEBVIEW_ALREADY_INITIALIZED);
+            }
+
+            if (WebViewInitializing)
+            {
+                // Cannot initialize WebView since it is already being initialized
+                throw new InvalidOperationException(DesignerUI.E_WEBVIEW_ALREADY_INITIALIZING);
+            }
+
+            _initializationState = InitializationState.IsInitializing;
+
+            base.BeginInit();
+        }
+
+        /// <inheritdoc />
+        public override void EndInit()
+        {
+            if (!WebViewInitializing)
+            {
+                // Cannot complete WebView initialization that is not being initialized
+                throw new InvalidOperationException(DesignerUI.E_WEBVIEW_NOT_INITIALIZING);
+            }
+
+            base.EndInit();
+        }
+
+        // Ensures this class is initialized. Initialization involves using ISupportInitialize methods
+        private void EnsureInitialized()
+        {
+            // If not already initialized and not already initializing
+            if (!WebViewInitialized && !WebViewInitializing)
+            {
+                BeginInit();
+                EndInit();
+            }
+        }
+
+        private delegate void PropertyInvalidator(WebView webViewHost);
+
+        /// <inheritdoc />
```

Comment:
Inheritdoc doesn't work for implicitly implemented members. This'll need a cref:
`<inheritdoc cref="IWebView.AcceleratorKeyPressed" />`
This goes for all the interface members here that aren't override from the baseclass

Reply:
I'm checking all the documentation

Reply:
InheritDoc does figure these out, but it fails to load the Gaze assembly (bug in InheritDoc) and fails to overwrite the XML documentation. I'm looking into a workaround now

Reply:
[Microsoft.Toolkit.Win32.UI.Controls.xml.txt](https://github.com/Microsoft/UWPCommunityToolkit/files/1964955/Microsoft.Toolkit.Win32.UI.Controls.xml.txt)

Reply:
@dotMorten I am able to see items like `AcceleratorKeyPressed` show up without any changes (the error for C++ went away after I did a `git clean -fdx`). What doesn't show up properly are items from a `System.*` namespace, which looks like a configuration issue I'll sort out. I've attached a sample output from the build process and verified the correct documents are being placed in the nupkg file (less the issue I described above).

Reply:
[Microsoft.Toolkit.Win32.UI.Controls.3.0.0-preview-build.207.ge3403d1b5d.zip](https://github.com/Microsoft/UWPCommunityToolkit/files/1965204/Microsoft.Toolkit.Win32.UI.Controls.3.0.0-preview-build.207.ge3403d1b5d.zip)